.flowconfig

```diff
@@ -13,4 +13,4 @@
 [options]
 module.name_mapper.extension='scss' -> 'empty/object'
 module.system=haste
-module.name_mapper='\(Applications\|Dashboard\|Navigation\|Onboarding\|Policies\|services\|Stats\|Types\|Users\|utilities\)\(.*\)$' -> '<PROJECT_ROOT>/app/javascript/src/\1/\2'
+module.name_mapper='\(Applications\|Dashboard\|Navigation\|Onboarding\|Policies\|services\|Stats\|Types\|Users\|utilities\|NewService\)\(.*\)$' -> '<PROJECT_ROOT>/app/javascript/src/\1/\2'
```

Comment:
But we have 2 forms: 1 for New Services (form scratch), and the other for Service Discovery (from Openshift)

Reply:
you might want to have just 1 form component, then those other different components will render it with their specifics

Reply:
I refactored to just one form component, but I still think the name shouldn't be `ServiceDiscovery`, because Service Discovery is just the half of the functionality.